tests/test_new_talk_voting.py

```diff
@@ -37,31 +63,164 @@ def test_talk_voting_available_with_proposal(user_client):
     response = user_client.get(url)
 
     assert response.status_code == 200
+    assert template_used(response, "ep19/bs/talk_voting/voting.html")
+
+
+@mock.patch('conference.talk_voting.is_user_allowed_to_vote', return_value=True)
+def test_talk_voting_lists_proposed_talks(mock_allowed_to_vote, user_client):
+    get_default_conference()
```

Comment:
here, we don't use the conference instance, why?

Reply:
Most views depend on a conference existing.